ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Consent.feature

```diff
@@ -10,6 +10,18 @@ Scenario: Individual participant consents to video hearing
 	When provides answer as Yes
 	When proceeds to next page
 	Then Individual should be on 'check your computer' screen
+	When provides answer as Yes
```

Comment:
It doesn't feel like this test maps to the scenario any more, should this entire test be on the `ThankYou` feature instead? And maybe have a different scenario text?

Reply:
Created a ThankYou feature for Individual and moved the scenario into it